docs/deployment.rst

```diff
@@ -0,0 +1,36 @@
+Safeguard
+---------
+
+To balance contention between topics, certain safeguard-limits are placed on each topic-partition pull.
+These limits are configurable. Please see configuration section. Below is an further explanation
+for how these safeguards work, and why they are needed. By default, all the safeguard configurations
+are turned off, you need to configure them explicitly in your Camus configuration file.
+
+Limiting timestamp duration: This can be set with ``kafka.max.pull.hrs``.
```

Comment:
There are 3 things you list here. Let's either call this out as separate subsections or as a numbered list